src/smart-components/OpenChannel/components/OpenChannelMessageList/__tests__/useHandleOnScrollCallback.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { useHandleOnScrollCallback, calcScrollBottom } from '../useHandleOnScrollCallback';
+import { renderHook } from '@testing-library/react'
+
+const prepareMockEvent = ({
+  scrollTop = 0,
+  scrollHeight = 0,
+  clientHeight = 0,
+}) => {
+  return {
+    target: {
+      scrollTop: scrollTop || 0,
+      scrollHeight: scrollHeight || 0,
+      clientHeight: clientHeight || 0,
+    },
+  } as unknown as React.UIEvent<HTMLDivElement, UIEvent>;
```

Comment:
TIL this, thanks 👍
This will be a good workaround for defining several mock data in testing code